resources/views/hardware/checkin.blade.php

```diff
@@ -74,7 +74,13 @@
                     </div>
 
                   @include ('partials.forms.edit.location-select', ['translated_name' => trans('general.location'), 'fieldname' => 'location_id', 'help_text' => ($asset->defaultLoc) ? 'You can choose to check this asset in to a location other than the default location of '.$asset->defaultLoc->name.' if one is set.' : null])
-
+                  <!-- Redirect to Asset Checkbox-->
+                    <div class="form-group">
+                      {{ Form::label('asset_redirect', trans('admin/hardware/form.asset_redirect'), array ('class' => 'col-md-3 control-label')) }}
+                      <div class="col-md-8">
+                        {{ Form::checkbox('asset_redirect', '1', false) }}
```

Comment:
We always want to be using the iCheck classes here - I believe you’ll want to add `minimal`, but I’m AFK so you can check the “remember me” checkbox on the login blade for the specific class. It should look larger and square, versus as a default checkbox.

Also I kinda feel like from a UI perspective, this option would make more sense in the box-footer, to the left of the submit button, but with distance from the cancel button. 

We should also set a cookie or setting here so that people don’t have to check it every time. (They will complain about it, I promise.)

An alternate approach would be something like a select box in that footer that offers:

- Return to Asset
- Go to User/Location/Whatever

It would default to the default current behavior (return to asset), but if they select something different, we set a cookie and make sure the intentionally selected option is pre-selected. Basically we “remember” whatever their last choice was.